core/controllers/incoming_emails.py

```diff
@@ -30,15 +28,14 @@ class IncomingReplyEmailHandler(base.BaseHandler):
     @acl_decorators.open_access
     def post(self, reply_to_id):
         incoming_mail = mail.InboundEmailMessage(self.request.body)
-        model = (
-            email_models.GeneralFeedbackEmailReplyToIdModel
-            .get_by_reply_to_id(reply_to_id))
+        feedbackReplyToId = email_services.get_general_feedback_reply_to_id(
```

Comment:
Actually, consider calling this feedback_thread_reply_info, and make similar changes to class names in the domain layer and other variable/method names. It is a bit misleading to call the object feedback_reply_to_id because reply_to_id is defined as a string [here](https://github.com/oppia/oppia/pull/5940/files#diff-70f75c4dfc82de12849ce872e69e4fdfR23) (so the original naming is unfortunate). We can't change the storage model easily but we can at least represent things properly in the domain layer.